modules/api_reference/src/Plugin/OpenApiValidationException.php

```diff
@@ -2,10 +2,12 @@
 
 namespace Drupal\devportal_api_reference\Plugin;
 
+use Exception;
```

Comment:
Yep, do not use classes from the global namespace even if PHPstorm tries to enforce you. Remember, we had a vote about this and `\Drupal`.